src/change/sdp.rs

```diff
@@ -1151,18 +1195,43 @@ impl From<MediaKind> for MediaType {
 }
 
 struct AsSdpParams<'a, 'b> {
-    pub candidates: &'a [Candidate],
-    pub creds: &'a IceCreds,
+    pub candidates: Cow<'a, [Candidate]>,
```

Comment:
@algesten you are going to be sad, but this seemed like the best solution, it still maintains the reference in most cases

Reply:
We could change our local candidates immediately on creating the offer I suppose